### DISCUSSION TITLE
change default number field value depending of a radio field.

### DISCUSSION THREAD

**Discussion by revolonl** [ansibleguy76/ansibleforms#131] [Q&A]
Hello,
i have a Radio field (Small-Medium-Large). I want to set a number field with a number depending of the radio choice (2 if small is selectected, 4 if it's Medium and 8 if it's large)
Is it possible, and how ?

Thanks.

**Comment by ansibleguy76** [comment]
not yet, for now the values will be medium/large.  but a local expression could fix this.  i'll convert this to an issue/improvement.  you could also use an enum field, there it is possible.

**Reply by revolonl** [reply]
Thanks !

**Reply by ansibleguy76** [reply]
added in 5.0.0 (latest beta now)

**Reply by ansibleguy76** [reply]
```
type: radio
values:
- label: small
  value: 2
- label: medium
  value: 4
```
